core/src/main/kotlin/com/malinskiy/marathon/execution/DevicePoolActor.kt

```diff
@@ -56,15 +57,15 @@ class DevicePoolActor(private val poolId: DevicePoolId,
         devices.filter {
             !it.value.isClosedForSend
         }.forEach {
-            it.value.send(DeviceEvent.WakeUp)
+            it.value.safeSend(DeviceEvent.WakeUp)
```

Comment:
If you use this safeSend then please delete devices.filter

Reply:
Fixed